target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -238,6 +238,9 @@ ramips_board_detect() {
 			name="linkits7688"
 		fi
 		;;
+	*"Widora NEO")
+		name="widora"
```

Comment:
Are you sure about the board name? Why not use ex. "widora-neo"? That would save you potential problems in future if you decide to make other board named ex. "Widora XYZ".